tools/clang/test/HLSLFileCheck/validation/completed-handle-all.ll

```diff
@@ -43,9 +43,11 @@ define void @loadStress_16() {
   ; test duplicate output complete call
   ; CHECK: error: Invalid use of completed record handle.
   ; CHECK: note: at 'call void @dx.op.outputComplete(i32 241, %dx.types.NodeRecordHandle %5)
+  ; CHECK: note: record handle invalidated by OutputComplete
+  ; CHECK: note: at 'call void @dx.op.outputComplete(i32 241, %dx.types.NodeRecordHandle %5)'
```

Comment:
Looks like an improvement, now if only we could print the instruction value assignment along with the call here, we could identify which call was being referred to!  Right now, it's not really helpful.

Reply:
I guess there's no value assignment because it returns void.  Is there any other way to uniquely identify this instruction in the stream?